RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -407,13 +389,16 @@ private string BuildPlugin(RoslynPluginDefinition definition, string baseTempDir
             string fullJarPath = Path.Combine(outputDirectory,
                 definition.Manifest.Key + "-plugin-" + definition.Manifest.Version + ".jar");
 
-            string repoKey = RepositoryKeyUtilities.GetValidKey(definition.PackageId + "." + definition.Language);
+            string repositoryId = RepositoryKeyUtilities.GetValidKey(definition.PackageId + "." + definition.Language);
+
+            string repoKey = RepositoryKeyPrefix + repositoryId;
 
-            RulesPluginBuilder builder = new RulesPluginBuilder(logger);
+            RoslynPluginJarBuilder builder = new RoslynPluginJarBuilder(logger);
```

Comment:
I created a new RoslynPluginJarBuilder class with a very similar API to the old RulesPluginBuilder to minimise the change in the caller. I'll probably rename and move the new class out of the Roslyn-specific assembly, as it doesn't contain any Roslyn-specific code.